src/OrchardCore.Commerce.Web/.eslintrc.json

```diff
@@ -0,0 +1,12 @@
+{
+  // Keep in sync with .eslint.global.json.
```

Comment:
Shouldn't we have such a file? Also, can't we have such a root, global _eslintrc and _stylelintrc_?

Reply:
This file just gets added when you reference Lombiq.NoteJs.Extensions. Presumably the _.eslint.global.json_ file exists there. I didn't do anything with _.eslintrc.json_ and _.stylelintrc_. I contemplated adding them to _.gitignore_, but I figured that could cause problems and confusion later if we actually need JS or CSS features in this project.

Reply:
I see. Can't we have such a root, global _eslintrc_ and _stylelintrc_ though? Or in any case, why just have these for the Web project, when we have assets in `OrchardCore.Commerce`?

Reply:
> Can't we have such a root, global eslintrc and stylelintrc though?

I don't know, ask @0liver. I still don't fully understand the mechanics.

> Or in any case, why just have these for the Web project, when we have assets in `OrchardCore.Commerce`?

[As I said](https://github.com/OrchardCMS/OrchardCore.Commerce/pull/139#discussion_r939727452), `Lombiq.NodeJs.Extensions` was already added to`OrchardCore.Commerce` from OCC-11.
You can see them in `main` [here](https://github.com/OrchardCMS/OrchardCore.Commerce/tree/main/src/Modules/OrchardCore.Commerce#:~:text=5%20days%20ago-,.eslintrc.json,-Temporarily%20make%20CI).

Reply:
Ah, of course. However, why do we need it for the Web project, since there are no assets there? Only for MD linting?

Reply:
The comment is somewhat misleading. I've added it only as documentation to the "template" files - they should really not end up in the consuming project.

And they won't after merging from `task/node-extensions` because Lukó moved that comment into a separate _Notes_ file [here](https://github.com/Lombiq/NodeJs-Extensions/pull/8/files#diff-a3555a89ac3d169a81fcb00074256c52af4331693730744c0eda2f9272b3d5acR3).

Reply:
> why do we need it for the Web project, since there are no assets there? Only for MD linting?

Yeah, since you specifically asked to do it this way in OSOCE. :wink: ([also](https://github.com/Lombiq/NodeJs-Extensions/blob/dev/Lombiq.NodeJs.Extensions/Docs/Markdown.md#:~:text=we%20suggest%20setting%20this%20value%20in%20the%20solution's%20%22entry%22%20project%20such%20as%20your%20web%20project))

Reply:
Yeah, I'm aware :). Can we get rid of the rc files though, since we don't need SCSS and JS linting here?

Reply:
Set
```xml
<NodeJsExtensionsGlobalESLintConfigurationDirectory></NodeJsExtensionsGlobalESLintConfigurationDirectory>
<NodeJsExtensionsCreateStylelintConfigurationFile></NodeJsExtensionsCreateStylelintConfigurationFile>
```
in the web project's .csproj file.

Reply:
This is missing.